poldeepner/core/poldeepner.py

```diff
@@ -2,17 +2,19 @@
 import os
 
 from wrapper import Sequence
+from embedding_wrapper import load_embedding
```

Comment:
This module is missing on repo

Reply:
Fixed, somehow I added it on another branch which was never uses